src/dojos/cd-dojo-details.vue

```diff
@@ -212,6 +227,12 @@
           },
         });
       },
+      volunteer(userType) {
+        service.requestUserInvite(this.user, this.id, userType);
+        /* eslint-disable no-alert */
+        alert('The Champion of this Dojo has been notified that you want to volunteer.');
```

Comment:
Should probably only alert after a successfully resolved promise..?

Reply:
async/await for this?

Reply:
won't the message still trigger if the API call is faulty ?

Reply:
Nope, if the API call rejects, an error will be thrown. If we really wanted to, we could do
```
/* eslint-disable no-alert */
try {
  await service.requestUserInvite(this.user, this.id, userType);
  alert('The Champion of this Dojo has been notified that you want to volunteer.');
} catch(o_O) {
  alert('Request failed because blah');
}
/* eslint-enable no-alert */
```

Reply:
And yes, I'm no longer doing `catch(e)`, it's `catch(o_O)` from now on 😛

Reply:
So which do we want? :)

Reply:
Done :)